lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -85,7 +87,7 @@ module.exports = {
               );
 
               if (!isAlreadyAccelerated) {
-                this.serverless.cli.log('Not using S3 Transfer Acceleration (1st deploy)');
+                legacy.log('Not using S3 Transfer Acceleration (1st deploy)');
```

Comment:
Do we have an agreement on not showing this information, even in verbose mode?

Reply:
no, in my opinion it should be shown in verbose mode that I wanted to address in separate PR, I'm going to revert the change here and cover both switch to `legacy.log` and new log in verbose mode